src/frontend/src/lib/components/hero/about/AboutHow.svelte

```diff
@@ -0,0 +1,16 @@
+<script lang="ts">
+	import AboutItem from '$lib/components/hero/about/AboutItem.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils';
+
+	export let asMenuItem = false;
+</script>
+
+<AboutItem openModal={modalStore.openAboutHow} {asMenuItem} on:click>
```

Comment:
`on:click={modalStore.openAboutHow}`

Reply:
in this case, I cannot bubble the on:click (it says `Duplicate attribute click`). So I would need to re-dispatch the event:

```typescript
const dispatch = createEventDispatcher();

const handleClick = (event: MouseEvent) => {
modalStore.openAboutHow();
dispatch('click', event);
};
```

not an issue, it is just that I would need to do the same both in `AboutWhat` and `AboutHow` components. Not really sure if I prefer it like that. Any strong opinion?

Reply:
Ah no I get what you are trying to do

```
on:click
on:click={openModal}
```

Maybe it works, but let's try to avoid it. Maybe you are but, I'm not exactly sure how Svelte behaves when handling two clicks and how safe it will be in the future?

Moreover, as we already discussed, even though we are in this weird place between Svelte 4 and 5, let's stick to bubbling events for consistency.

```
const doSomething = () => {
   // do something if needed
   dispace("custmoEvent")
}

on:click={doSomething}
```

Agree?

Reply:
ah ok! so re-dispatching it is! on it